docs/standards/universal-profile/lsp6-key-manager.md

```diff
@@ -335,7 +364,7 @@ Same as `STATICCALL`, but allowing to interact with any contract. This will not
         <b>value = </b><code>0x0000000000000000000000000000000000000000000000000000000000004000</code>
     </p>
 
-Same as `DELEGATECALL`, but allowing to interact with any contract. This will not check for [**Allowed Calls**](#allowed-calls) if the caller has any of these restrictions set.
+Same as `DELEGATECALL`, but allowing to interact with any contract. This will not check for [**Allowed Calls**](#allowed-calls) if the controller has any of these restrictions set.
```

Comment:
```suggestion
Same as `DELEGATECALL`, but allows interacting with any contract. This will not check if the controller has any  [**Allowed Calls**](#allowed-calls) restrictions set.
```